ANMN/LTSP/TSaggregator/TSagg_metadata.json

```diff
@@ -258,33 +299,69 @@
+        "keywords_vocabulary": "IMOS parameter names. See https://github.com/aodn/imos-toolbox/blob/master/IMOS/imosParameters.txt",
         "license": "http://creativecommons.org/licenses/by/4.0/",
         "naming_authority": "IMOS",
         "project": "Integrated Marine Observing System (IMOS)",
-        "references": "http://www.imos.org.au", 
-        "source": "Mooring", 
+        "references": "http://www.imos.org.au",
+        "source": "Mooring",
         "standard_name_vocabulary": "NetCDF Climate and Forecast (CF) Metadata Convention Standard Name Table 45",
-        "lineage": "The variable of interest (VoI) is produced by sequentially concatenating the individual values in each of the input files. The resulting variable has dimension OBSERVATION. The VoI's ancillary_variables, in particular the corresponding quality-control flags, are also included, with dimension OBSERVATION. If the quality control variable is absent in any input file, the corresponding flags in the output file will be set to 0 ('no QC performed'). The variable TIME from input files is concatenated into a variable TIME(OBSERVATION). This could result in a non-uniform time interval. The DEPTH variable from input files is concatenated into a variable DEPTH(OBSERVATION). If not present, fill values are stored. DEPTH_quality_control, if present, is also included. Where  the DEPTH variable is absent, the corresponding DEPTH_quality_control values are set to 9 ('missing value'). The variables PRES (sea_water_pressure) and PRES_REL (sea_water_pressure_due_to_sea_water) are aggregated in exactly the same way as DEPTH. All output variables with the INSTRUMENT dimension are sorted in chronological order. In order to keep track of the provenance of VoI in the aggregated file, accessory variables are created. "
+        "lineage": "The variable of interest (VoI) is produced by sequentially concatenating the individual values in each of the input files. The resulting variable has dimension OBS. The VoI's ancillary_variables, in particular the corresponding quality-control flags, are also included, with dimension OBS. If the quality control variable is absent in any input file, the corresponding flags in the output file will be set to 0 ('no QC performed'). The variable TIME from input files is concatenated into a variable TIME(OBS). This could result in a non-uniform time interval. The DEPTH variable from input files is concatenated into a variable DEPTH(OBS). If not present, fill values are stored. DEPTH_quality_control, if present, is also included. Where  the DEPTH variable is absent, the corresponding DEPTH_quality_control values are set to 9 ('missing value'). All output variables with the OBS dimension are sorted in chronological order. In order to keep track of the provenance of VoI in the aggregated file, accessory variables are created. Product created with https://github.com/aodn/..."
```

Comment:
It looks like you've reverted the value of `lineage` to an earlier version (it still talks about OBS dimension, and has the partial code URL at the end...) I assume that was unintended.

Reply:
yeah.. copypaste error... fixed/